server/daos/booksLanguages.js

```diff
@@ -0,0 +1,41 @@
+import db from '@database/models';
+import { isEmpty } from 'lodash';
+
+export const insertBooksLanguages = args => {
+  if (!isEmpty(args.booksLanguages)) {
+    const mapBooksLanguagesArgs = args.booksLanguages.map((item, index) => ({
+      bookId: args.bookId,
+      languageId: item.languageId
+    }));
+
+    return db.booksLanguages.bulkCreate(mapBooksLanguagesArgs);
```

Comment:
hmm this doesn't seem right. We can't not do anything in the else case. the dao should receive things in a formatted manner. Formatting shouldn't happen in the DAO